bot/exts/utilities/epoch.py

```diff
@@ -0,0 +1,82 @@
+import discord
+from discord.ext import commands
+from discord.ext.commands import Converter
+from bot.bot import Bot
+from typing import Union
+
+import dateutil
+from dateutil import parser
+import arrow
+
+# https://discord.com/developers/docs/reference#message-formatting-timestamp-styles
+TIMESTAMP_FORMATS = ["H:mm A", "H:mm:ss A", "MM/DD/YYYY", "MMMM D, YYYY", "MMMM D, YYYY H:mm A",
+                     "dddd, MMMM D, YYYY hh:mm A "]
+STYLES = {"Epoch": "", "Short Time": "t", "Long Time": "T", "Short Date": "d", "Long Date": "D", "Short Date/Time": "f",
```

Comment:
```suggestion
STYLES = {"Epoch": "f", "Short Time": "t", "Long Time": "T", "Short Date": "d", "Long Date": "D", "Short Date/Time": "f",
```
Removes the need for a check, (this being the default value)